Client/Frontend/Browser/URIFixup.swift

```diff
@@ -10,8 +10,8 @@ class URIFixup {
         if let url = URL(string: entry), InternalURL.isValid(url: url) {
             return URL(string: entry)
         }
-
-        let trimmed = entry.trimmingCharacters(in: .whitespacesAndNewlines)
+        let replaced = replaceBrackets(str: entry)
+        let trimmed = replaced.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
Can you move this to after `trimmed.addingPercentEncoding`? 
Then the code becomes `guard var escaped ...` and then `escaped = replaceBrackets...`

Reply:
Done!